app/assets/stylesheets/parent_objects.scss

```diff
@@ -88,9 +96,14 @@ DEFAULT DESKTOP STYLING
   padding-top: 10px;
 }
 
+.key {
+  width: 15%;
+}
+
 #parent-show {
-  display: grid;
-  grid-template: 'a b' auto;
+  width: 100%;
+  padding-top: 3%;
+  display: flex;
 }
 
 #parent-show em {
```

Comment:
This can be deleted as well because the 'grid area' is dependent on the 'grid template' that got removed.